bqskit/ir/opt/multistartgens/naive.py

```diff
@@ -0,0 +1,63 @@
+"""This module implements the NaiveStartGenerator base class."""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+from bqskit.ir.opt.multistartgen import MultiStartGenerator
+from bqskit.utils.typing import is_integer
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
+    from bqskit.ir.circuit import Circuit
+    from bqskit.qis.state.state import StateVector
+    from bqskit.qis.unitary.unitarymatrix import UnitaryMatrix
+
+
+class NaiveStartGenerator(MultiStartGenerator):
```

Comment:
I don't think this is very Naive, maybe we should call it DiagonalStartGenerator instead? Thoughts?

Reply:
Sure! I mean it was naive in the sense that it was an educated guess of how to improve multistart. But I think diagonal is much more descriptive.